test/claim.test.ts

```diff
@@ -9,12 +9,14 @@ describe('Tests for claim operation', () => {
 
     beforeEach(async () => {
         sdk = new MockRelayingServices();
+        await sdk.initialize({});
     });
 
     it('Should fail by not implemented error', async () => {
         try {
             await sdk.claim({});
-        } catch (error) {
+            fail('The claim will not implement in this version of SDK');
```

Comment:
Thanks for adding the fail statement. I would specify also, that it shouldn't have occurred.
```suggestion
            fail("expected claim to fail, but it didn't");
```

Reply:
Fixed on the commit https://github.com/infuy/relaying-services-sdk/pull/3/commits/22632b07bd72e5d1f295c8019761e09cca1b0a06.